github/src/main/scala/org/typelevel/sbt/TypelevelGitHubPlugin.scala

```diff
@@ -82,19 +82,24 @@ object TypelevelGitHubPlugin extends AutoPlugin {
       val userRepo = gitHubUserRepo.value
       val infoOpt = scmInfo.value
 
-      if (tlIsScala3.value)
-        tagOrHash.toSeq flatMap { vh =>
-          userRepo.toSeq flatMap {
-            case (user, repo) => Seq(s"-source-links:github://${user}/${repo}", "-revision", vh)
-          }
-        }
-      else
-        tagOrHash.toSeq flatMap { vh =>
-          infoOpt.toSeq flatMap { info =>
-            val path = s"${info.browseUrl}/blob/${vh}€{FILE_PATH}.scala"
-            Seq("-doc-source-url", path)
-          }
+      tagOrHash.toSeq flatMap { vh =>
+        scalaVersion.value match {
+          case V(V(3, _, _, _)) =>
+            userRepo.toSeq flatMap {
+              case (user, repo) =>
+                Seq(s"-source-links:github://${user}/${repo}", "-revision", vh)
+            }
+          case (V(V(2, minor, patch, _))) =>
+            infoOpt.toSeq flatMap { info =>
+              val path =
+                // see https://github.com/scala/bug/issues/12867#issuecomment-1718481858
+                if (minor > 13 || minor == 13 && patch.exists(_ >= 12))
+                  s"${info.browseUrl}/blob/${vh}/€{FILE_PATH}.scala"
+                else s"${info.browseUrl}/blob/${vh}€{FILE_PATH}.scala"
```

Comment:
Dumb question: what if we just always put the slash? Does it work with 2.12, old 2.13s?

Reply:
I wondered about this, then I didn't want to play with 🔥

Reply:
Conclusion is, while it does double the slash in the link, this _appears_ to be benign and the link still gets to github ok. However, that feels dodgy to me